StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/GenericInfoScreen/GenericInfoViewController.swift

```diff
@@ -49,7 +49,23 @@ final class GenericInfoViewController: SheetViewController {
                 }(),
                 title: genericInfoScreen.header?.title,
                 subtitle: genericInfoScreen.header?.subtitle,
-                contentView: nil, // TODO(kgaidis): add support for content view
+                headerAlignment: {
+                    let headerAlignment = genericInfoScreen.header?.alignment
+                    switch headerAlignment {
+                    case .center:
+                        return .center
+                    case .right:
+                        return .trailing
+                    case .left: fallthrough
```

Comment:
And this be `case .left: return .leading`?

Reply:
similar question as this:

https://github.com/stripe/stripe-ios/pull/3767/files/13013419856f14f62a2f1834ea2ebd21983593f2#r1676488006